etc/kayobe/environments/ci-aio/inventory/hosts

```diff
@@ -7,3 +7,6 @@ controllers
 
 [monitoring:children]
 controllers
+
+[network:children]
+controllers
```

Comment:
We already have this in etc/kayobe/inventory/groups

Reply:
The problem is the `delegate_to: network`. delegate_to expects a single host, not a group. But really we want to remove the file from all hosts in the group. I'd suggest making a separate play that targets the network group.